README.md

```diff
@@ -142,7 +150,7 @@ discussion of the primary options.
 
 Deployment can be done by using the following single command:
 ```shell
-helm install . --namespace=openwhisk --name=owdev -f mycluster.yaml
+helm install helm --namespace=openwhisk --name=owdev -f mycluster.yaml
```

Comment:
I will suggest to use "./helm" to indicate the second `helm` is a chart directory. So that users know the correct place to run this command.

Reply:
done. thanks.